sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python
+
+# Copyright 2019 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import rospy
+import roslaunch
+import rospkg
+import Tkinter as tk
+import tkMessageBox as messageBox
+import paramiko
+import sys
+import yaml
+from rosparam import upload_params
+
+
+class SrUrLoadCalibrationExceptions(Exception):
+    pass
+
+
+class NoArmsSpecified(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class ArmTypeNotRecognised(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class DifferentArmTypes(SrUrLoadCalibrationExceptions):
+    pass
+
+
+class SrUrLoadCalibration(object):
+    def __init__(self, arm_info_in=[]):
+        self._default_kinematics_config = ''
+        self._arm_calibrations_folder = ''
+        self._CONST_UR_ARM_SSH_USERNAME = "root"
+        self._CONST_UR_ARM_SSH_PASSWORD = "easybot"
+        try:
+            if [] == arm_info_in:
+                raise NoArmsSpecified("No arms specified, cannot find arm calibration")
+        except NoArmsSpecified as err:
+                rospy.logerr(err)
+        self._arm_info_in = arm_info_in
+        CONST_ARM_TYPE = self._arm_info_in[0]['arm_type'].lower()
+        try:
+            if len(self._arm_info_in) > 1:
+                for arm in self._arm_info_in:
+                    if arm['arm_type'].lower() != CONST_ARM_TYPE:
+                        raise DifferentArmTypes("Different arm types specified. \
+                                                Currently this node only supports arms of the same type")
+        except DifferentArmTypes as err:
+            rospy.logerr(err)
+            
+        if 'sr_ur_calibration' in rospkg.RosPack().list():
+            CONST_SR_UR_ARM_CALIBRATION_ROOT = rospkg.RosPack().get_path('sr_ur_calibration')
+        else:
+            CONST_SR_UR_ARM_CALIBRATION_ROOT = rospkg.RosPack().get_path('sr_ur_arm_calibration_loader')
+        self._arm_calibrations_folder = os.path.join(CONST_SR_UR_ARM_CALIBRATION_ROOT, 'calibrations')
+        self._setup_folders()
+        if 'ur10e' or 'ur5e' or 'ur3e' or 'ur16e' in CONST_ARM_TYPE:
```

Comment:
this is not correct python syntax. You can check by trying something like:
asd = [0, 1, 2, 3, 4]
5 or 6 in asd

will just print 5, not False